src/include/cpu/rpp_cpu_simd.hpp

```diff
@@ -2865,10 +2865,17 @@ inline void rpp_store24_f32pln3_to_u8pkd3_avx(Rpp8u* dstPtr, __m256* p)
     _mm256_storeu_si256((__m256i *)(dstPtr), px1);          /* store the 24 U8 pixels in dst */
 }
 
+inline void rpp_store8_u8pln1_to_u8pln1_avx(Rpp8u* dstPtr, __m256i &p)
+{
+    __m128i pTemp = _mm256_castsi256_si128(p);
+    rpp_storeu_si64((__m128i *)(dstPtr), pTemp);
+}
+
 inline void rpp_store8_f32pln1_to_u8pln1_avx(Rpp8u* dstPtr, __m256 &p)
 {
     __m256i px1 = _mm256_permute4x64_epi64(_mm256_packus_epi32(_mm256_cvtps_epi32(p), avx_px0), _MM_SHUFFLE(3,1,2,0));
-    _mm256_storeu_si256((__m256i *)(dstPtr), _mm256_packus_epi16(px1, avx_px0));
```

Comment:
why is this change reverted?

Reply:
on rechecking understood that this change is essential, confirmed no other existing kernel fails due to this change